### ISSUE TITLE
Issue: Fail to send records from ButterflyCount app to website. Samsung galaxy

### ISSUE THREAD

**Issue by CrisSevilleja** [BiologicalRecordsCentre/ABLE#562]
### Describe the issue

Hello @kazlauskis 

One app user (Indicia Warehouse User ID 293904) had issues uploading records from ButterflyCount app, for transect and 15min count surveys. He got the message SORRY, FAILURE TO RECOVER.

He uninstalled the app and reinstalled it even with problems in sending, after logging out he was able to send the transect visit. He lost some 15min counts and was able to include the transect visit manually on the website. He is afraid of using the 15min counts and not able to upload them. 

He will continue trying. The issue is solved but I wanted to create the issue to report this problem with the Samsung galaxy in case there are more issues with this device or model.

### Expected behaviour

Be able to upload the app records to the website

### To Reproduce

1. Do a survey, transect or 15min count
2. Finish the survey
3. Click on the upload button 

### App Version

1.19.0 (226)

### Device

Samsung Galaxy model SM-510F 2016, Android 7.0.

### When did this happen

since April 2023

**Comment by kazlauskis** [comment]
It would be good to know the exact date-time when this has happened. It is the first time I see such an error code, and it isn't originating from the app. So it could be that, for example, that day the warehouse or the website was under maintenance. It is hard to say what has happened without finding the issue in the server logs.

**Comment by CrisSevilleja** [comment]
Hi Karolis, asking the user I got this answer:

"The 15 min count that I had to discard was done on 22 April, and the transect on the 24th. I don't remember if I tried to send it on the 22nd, but I'm sure I couldn't send it on the 24th. Sometimes I don't send them on the same day because I have doubts about some species. The second time it failed was on the 3rd of May (after logging out and deleting and re-downloading the app). Although, as I told you, I was finally able to do it after logging out again." 

I think this helps a little bit more. Thanks in advance
